modules/calc/src/main/java/com/opengamma/strata/calc/runner/CalculationParameters.java

```diff
@@ -47,7 +55,7 @@
    * The parameters, keyed by query type.
    */
   @PropertyDefinition(validate = "notNull")
-  private final ImmutableMap<Class<?>, CalculationParameter> parameters;
+  private final ImmutableMap<Class<? extends CalculationParameter>, CalculationParameter> parameters;
```

Comment:
While accepting a wildcard on input is useful, there is no value in declaring the local variable that way.